test/fixtures/box/cornerRadius.js

```diff
@@ -26,7 +26,7 @@ module.exports = {
               yMax: 10,
               backgroundColor: 'rgba(33, 101, 171, 0.5)',
               borderColor: 'rgb(33, 101, 171)',
-              cornerRadius: 8,
+              cornerRadius: {topLeft: 16, topRight: 8, bottomRight: 4},
```

Comment:
What happened to the purple annotation in this test? the new image appears to be missing it

Reply:
No idea, need to take a look.

Reply:
It was caused by unclamped infinite value